DataConnectors/Forcepoint Cloud Security Gateway.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Data connector json file should be in the format ProviderNameApplianceName.json . Please remove the space from DataConnector file name.